playground/backend/internal/utils/common_test.go

```diff
@@ -28,6 +28,7 @@ const (
    // CHANGE 1/3: Select a Beam runner, such as BlockingDataflowRunner\n    // or FlinkRunner.\n    // CHANGE 2/3: Specify runner-required options.\n    // For BlockingDataflowRunner, set project and temp location as follows:\n    //   DataflowPipelineOptions dataflowOptions = options.as(DataflowPipelineOptions.class);\n    //   dataflowOptions.setRunner(BlockingDataflowRunner.class);\n    //   dataflowOptions.setProject(\"SET_YOUR_PROJECT_ID_HERE\");\n    //   dataflowOptions.setTempLocation(\"gs://SET_YOUR_BUCKET_NAME_HERE/AND_TEMP_DIRECTORY\");\n    // For FlinkRunner, set the runner as follows. See {@code FlinkPipelineOptions}\n    // for more details.\n    //   options.as(FlinkPipelineOptions.class)\n    //      .setRunner(FlinkRunner.class);\n\n    // Create the Pipeline object with the options we defined above\n    Pipeline p = Pipeline.create(options);\n\n    // Concept #1: Apply a root transform to the pipeline; in this case, TextIO.Read to read a set\n    // of input text files. TextIO.Read returns a PCollection where each element is one line from\n    // the input text (a set of Shakespeare's texts).\n\n    // This example reads from a public dataset containing the text of King Lear.\n    p.apply(TextIO.read().from(\"gs://apache-beam-samples/shakespeare/kinglear.txt\"))\n\n        // Concept #2: Apply a FlatMapElements transform the PCollection of text lines.\n        // This transform splits the lines in PCollection<String>, where each element is an\n        // individual word in Shakespeare's collected texts.\n        .apply(\n            FlatMapElements.into(TypeDescriptors.strings())\n                .via((String line) -> Arrays.asList(line.split(\"[^\\\\p{L}]+\"))))\n        // We use a Filter transform to avoid empty word\n        .apply(Filter.by((String word) -> !word.isEmpty()))\n        // Concept #3: Apply the Count transform to our PCollection of individual words. The Count\n        // transform returns a new PCollection of key/value pairs, where each key represents a\n        // unique word in the text. The associated value is the occurrence count for that word.\n        .apply(Count.perElement())\n        // Apply a MapElements transform that formats our PCollection of word counts into a\n        // printable string, suitable for writing to an output file.\n        .apply(\n            MapElements.into(TypeDescriptors.strings())\n                .via(\n                    (KV<String, Long> wordCount) ->\n                        wordCount.getKey() + \": \" + wordCount.getValue()))\n        // Concept #4: Apply a write transform, TextIO.Write, at the end of the pipeline.\n        // TextIO.Write writes the contents of a PCollection (in this case, our PCollection of\n        // formatted strings) to a series of text files.\n        //\n        // By default, it will write to a set of files with names like wordcounts-00001-of-00005\n        .apply(TextIO.write().to(\"wordcounts\"));\n\n    p.run().waitUntilFinish();\n  }\n}"
```

Comment:
Could you use https://pkg.go.dev/embed to replace this large blob of text?

Reply:
It's possible to use directly from file